app/views/purchases/index.html.erb

```diff
@@ -0,0 +1,30 @@
+<div class="header-1">
+  <div class="back">
+    <a href="/categories" class="back-link"><span class='back-arrow'> &#8592; </span></a>
+  </div>
+  <div class="middle-title">
+    <center>
+      <h1>Transactions</h1>
+      <p>
+        Category <%= @category_name %> .
+        Total $<%= current_user.purchases.joins(:categories).where(categories: { id: params[:category_id] }).sum(:amount) %> 
+      </p>
+    </center>
+  </div>
+  <div class="next">
+    &nbsp;
+  </div>
+</div>
+
+<div class="categories">
+  <center>
+    <p style="color: green"><%= notice %></p>
+  </center>
+  <br />
+  <% @purchases.each do |purchase| %>
+    <%= render purchase %>
+  <% end %>
+  <center>
+    <%= link_to "New purchase", new_purchase_path, class: 'down-btn' %>
+  </center>
```

Comment:
- [ ] Same here, As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=size%20and%20weight.-,Layout%3A%20composition%20and%20space%20between%20elements.,-NOTE%3A%20In%20these), Please add the correct placement for this button at the end of the page and make it take the whole page width to match the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/228712936-e3e144b2-6ed8-4ed0-941f-1d47b976490a.png)  | ![image](https://user-images.githubusercontent.com/68796232/228713056-bd1a212f-4d20-49a5-ac88-1726e583ae70.png) |